File: Tasks/Task1/Solution/mainClass.cpp
#include <iostream>
#include "cpp\apartment.h"
#include "cpp\building.h"
#include "cpp\rooms.h"

using namespace std;

int main()
{

    string name = "Buildinh One";
    bool e = false;
    std::vector<Rooms> r = {Rooms(10, 10), Rooms(10, 15)};
    std::vector<Apartment> a = {Apartment(2, r)};
    Building b(name, e, a, r);

    b.displayInfo();

    return 0;
}

File: Tasks/Task1/Solution/main_class.py
from py.rooms import Rooms
from py.apartment import Apartment
from py.building import Building


r1 = Rooms(5, 10)
r2 = Rooms(7, 12)

if __name__ == "__main__":
    b = Building(
        "b1",
        True,
        Apartment(3, 2, [r1, r2]),
        [r1, r2],
    )

    b.display()
